src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/requestaccess/CreateResearchProjectStep1.java

```diff
@@ -44,6 +44,7 @@ public CreateResearchProjectStep1(CreateResearchProjectWizardStep1View view, Dat
 	public void configure(ManagedACTAccessRequirement ar, RestrictableObjectDescriptor targetSubject) {
```

Comment:
User-facing requestor UI.  For now, only show the IDU text box if an IDU is required.  And don't try to validate the IDU value if it's optional.